### ISSUE TITLE
feat [cli]: `readyWebhook` 

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#1773]
Allow users to set a custom webhook to fire when the API has successfully and completely been launched

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.2.2](https://github.com/open-wa/wa-automate-nodejs/compare/4.2.1...4.2.2) (2021-06-09)

- build(deps-dev): bump @typescript-eslint/eslint-plugin from 4.26.0 to 4.26.1 [`#1768`](https://github.com/open-wa/wa-automate-nodejs/pull/1768)
- build(deps-dev): bump @typescript-eslint/parser from 4.26.0 to 4.26.1 [`#1770`](https://github.com/open-wa/wa-automate-nodejs/pull/1770)
- Release 4.2.2 [`613907e`](https://github.com/open-wa/wa-automate-nodejs/commit/613907eb575f61305723d4faf179a66299d07692)
- ✨ CLI: implemented `readyWebhook`/`--ready-webhook` #1773 [`73d1d12`](https://github.com/open-wa/wa-automate-nodejs/commit/73d1d12ac608f32a8a7666e9e52a5e377a5b98c0)
